sqlmesh/core/scheduler.py

```diff
@@ -329,29 +331,23 @@ def _dag(self, batches: SnapshotToBatches) -> DAG[SchedulingUnit]:
         }
 
         dag = DAG[SchedulingUnit]()
+        dummy_node = ((to_datetime(0), to_datetime(0)), -1)
```

Comment:
should we call this `terminal_node` or `batch_end_node` to make its purpose clear?